src/entity.c

```diff
@@ -412,12 +414,12 @@ ecs_type_t copy_from_prefab(
         }
 
         ecs_table_column_t *src_column = &prefab_columns[p + 1];
-        uint32_t size = src_column->size;
+        uint64_t size = src_column->size;
```

Comment:
This would a bit excessive for a column (component type) size. The size in the type itself is `uint16_t`, I would use the same here.